README.md

```diff
@@ -145,7 +145,7 @@ source tree. For more detailed instructions please read [BuildingWithCMake.md](d
     cd build
 
     # Generate the build system using cmake
-    cmake ..
+    cmake -Wdev -C../cmake/caches/Travis.cmake  ..
```

Comment:
nit: The [documentation for cmake](https://cmake.org/cmake/help/latest/manual/cmake.1.html) suggests `-C` and the path are expected to be separate command-line arguments.

Reply:
Oh, I guess I just didn't notice that.

Reply:
Fixed.